packages/components/src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,192 @@
+     */
+    disabled,
+    /**
+     * Click callback for this item.
+     */
+    onClick,
+    /**
+     * A string value for the accordion item. All items within an accordion should use a unique value.
+     */
+    value,
+}: AccordionItemProps) => {
+    return (
+        <RadixAccordion.Item
+            className={styles.accordionItem}
+            value={value}
+            onPointerDown={(event) => {
+                event.currentTarget.dataset.showFocusRing = 'false';
+            }}
+            onBlur={(event) => {
+                event.currentTarget.dataset.showFocusRing = 'true';
+            }}
+            onClick={onClick}
+            disabled={disabled}
+            data-test-id={dataTestId}
+        >
+            {children}
+        </RadixAccordion.Item>
+    );
+};
+AccordionItem.displayName = 'Accordion.Item';
+
+export type AccordionHeaderProps = {
+    /**
+     * Children of the Accordion header. This should contain `Accordion.Trigger`
+     */
+    children?: ReactNode;
+};
+
+export const AccordionHeader = ({ children }: AccordionHeaderProps) => {
+    return <RadixAccordion.Header>{children}</RadixAccordion.Header>;
+};
+AccordionHeader.displayName = 'Accordion.Header';
+
+export type AccordionTriggerProps = { children?: ReactNode; 'data-test-id'?: string };
+
+export const AccordionTrigger = ({
+    'data-test-id': dataTestId = 'fondue-accordion-trigger',
+    /**
+     * Children of the Accordion trigger. This contains the actually clickable and visible header content
+     */
+    children,
+}: AccordionTriggerProps) => {
+    return (
+        <RadixAccordion.Trigger className={styles.accordionTrigger} data-test-id={dataTestId}>
+            {children}
+            <IconCaretDown className={styles.accordionCaret} size="16" />
+        </RadixAccordion.Trigger>
+    );
+};
+AccordionTrigger.displayName = 'Accordion.Trigger';
+
+type AccordionContentProps = {
+    'data-test-id'?: string;
+    /**
+     * Children of the Accordion content. This contains the main content.
+     */
+    children?: ReactNode;
+    /**
+     * Adds a divider line between the header and the content.
+     */
+    divider?: boolean;
+    /**
+     * Click callback for the content.
+     */
+    onClick?: MouseEventHandler<HTMLDivElement>;
+    /**
+     * Controls if we show paddings around the content.
+     * @default 'large'
+     */
+    padding?: AccordionPadding;
+};
+
+export const AccordionContent = ({
+    'data-test-id': dataTestId = 'collapsible-wrap',
+    children,
+    divider,
+    onClick,
+    padding = 'large',
+}: AccordionContentProps) => {
+    return (
+        <RadixAccordion.Content className={styles.accordionContent} onClick={onClick} data-test-id={dataTestId}>
+            <div
+                className={cn([
+                    styles.accordionContentText,
+                    styles[`accordionContentTextPadding-${padding}`],
+                    divider && styles.accordionContentTextDivider,
```

Comment:
data attr on the root element and focus via CSS instead of rerendering the full content on change?